### DISCUSSION TITLE
Unexpected https behavior

### DISCUSSION THREAD

**Discussion by lanyi1998** [projectdiscovery/httpx#1304] [Q&A]
I want to access http

https was accessed automatically.

<img width="679" alt="image" src="https://github.com/projectdiscovery/httpx/assets/31091395/ff4a40d7-d309-438d-a9c1-5af9ff7befc9">

The target did not issue a 301 response

<img width="1350" alt="image" src="https://github.com/projectdiscovery/httpx/assets/31091395/9298603c-5f69-407e-b042-20aac9bd2543">

**Comment by ehsandeep** [comment]
@lanyi1998 you can use `-nfs` option to force using input scheme.

```console
echo http://www.7k7k.com | httpx -nfs
    __    __  __       _  __
   / /_  / /_/ /_____ | |/ /
  / __ \/ __/ __/ __ \|   /
 / / / / /_/ /_/ /_/ /   |
/_/ /_/\__/\__/ .___/_/|_|
             /_/

		projectdiscovery.io

http://www.7k7k.com
```

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
